client/src/components/Register.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+
+const Register = () => (
+  <Grid container item direction='column' alignItems='center' spacing={3} style={{marginTop: '1rem'}}>
+
+    <Grid item>
+      <Typography variant='h2' align='center'>New User</Typography>
+    </Grid>
+
+    <Grid item>
+      <form action='http://localhost:5000/api/users/register' method='post'>
```

Comment:
One suggestion: you can add a 'proxy' field to your package.json like this :
 "proxy": "http://localhost:5000" 
Then anytime you need to send a request to your api, you can just do "/api/...." and it will automatically proxy your request to localhost:5000